packages/notebook/src/windowing.ts

```diff
@@ -52,27 +52,58 @@ export class NotebookViewModel extends WindowedListModel {
   estimateWidgetSize = (index: number): number => {
     const model = this.cells[index].model;
     const height = this.cellsEstimatedHeight.get(model.id);
+
+
     if (typeof height === 'number') {
       return height;
     }
 
+
     const nLines = model.sharedModel.getSource().split('\n').length;
     let outputsLines = 0;
+
     if (model instanceof CodeCellModel) {
+      const supportedOutputTypes = [
+        'text/html',
+        'image/svg+xml',
+        'application/pdf',
+        'text/markdown',
+        'text/plain',
+        'application/vnd.jupyter.stderr',
+        'application/vnd.jupyter.stdout',
+        'text'
```

Comment:
```suggestion
        'text/markdown',
        'text/plain',
        'application/vnd.jupyter.stderr',
        'application/vnd.jupyter.stdout',
        'text'
```

Maybe we should stick to only these as estimates from html/xml code would be likely unreliable?

Reply:
@krassowski  on it